x-pack/plugins/files/server/routes/file_kind/integration_tests/file_kind_http.test.ts

```diff
@@ -120,4 +120,84 @@ describe('File kind HTTP API', () => {
     } = await request.get(root, `/api/files/files/${fileKind}/list?page=1&perPage=5`).expect(200);
     expect(files2).toHaveLength(5);
   });
+
+  const twoDaysFromNow = (): number => Date.now() + 2 * (1000 * 60 * 60 * 24);
+
+  test('share', async () => {
```

Comment:
nit: it is nice when test title explains what the test does:

```suggestion
  test('returns a share token, when a file is successfully shared', async () => {
```

Reply:
I was thinking of these tests as simple tests of the endpoints so I named them just "upload", "download", or in this case "share". Happy to go with your suggestion though!